compiler/cbmc/src/goto_program/stmt.rs

```diff
@@ -207,6 +207,10 @@ impl Stmt {
         )
     }
 
+    pub fn assume_false(loc: Location) -> Self {
```

Comment:
doc comment

Reply:
For sure!

Reply:
I'm actually removing this since I am adding a builtin abort()